src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxNumber = 100;
+
+    public Ball getRandomBall() {
+        Random rnd = new Random();
+        int number = rnd.nextInt(maxNumber);
+        String color = ColorSupplier.getRandomColor();

```

Comment:
Use 'ColorSupplier.getRandomColor().name()' instead of 'ColorSupplier.getRandomColor()' to get the String representation of the enum constant. This ensures that you are using the enum's name rather than relying on the 'toString()' method, which could be overridden.